pkg/commands/server/pluginshealth.go

```diff
@@ -14,13 +13,21 @@ const (
 	// pluginsHealthCmdUsage is the usage text for the 'plugins health' command.
 	pluginsHealthCmdUsage = "Get a list of plugins' health that are configured with Synse Server"
 
+	// pluginsHealthCmdArgsUsage is the argument usage for the `plugins health` command.
+	pluginsHealthCmdArgsUsage = "[PLUGIN TAG]"
+
 	// pluginsHealthCmdDesc is the description for the 'plugins health' command.
 	pluginsHealthCmdDesc = `The plugins health command hits the active Synse Server host's '/plugins'
-  endpoint, returns health information of all configured plugins.
+  endpoint. If a plugin is provided, the CLI will return its health information.
```

Comment:
instead of just saying "if a plugin is provided", should be more specific and say "if a plugin tag is provided"